lib/rate_limiter.rb

```diff
@@ -55,7 +56,7 @@ def can_perform?
     rate_unlimited? || is_under_limit?
   end
 
-  def seconds_to_wait(now)
+  def seconds_to_wait(now = Time.now.to_i)
```

Comment:
I'm not sure if this default is correct since we do not ever want someone to be waiting for the number of seconds since Epoch?

Reply:
I guess the method name here is not perfect 😄 This method returns the number of seconds to wait _from_ the give time. So if `seconds_to_wait(now)` returns 60, then `seconds_to_wait(now - 1.minute)` returns 120.